### PR TITLE
BaSP-MR-46: SuperAdmin List

### PR DESCRIPTION
**Pull request by Dannylez**
SuperAdmin List with funcionalities (GET, PUT, POST, DELETE)

### CODE REVIEW

**Comment by Dannylez** [comment]
[SuperAdmins - LIST Add/Edit/Delete](https://trello.com/c/G69QR8X6/46-superadmins-list-add-edit-delete)

**Review by DelfiLT — APPROVED** [review]

File: src/Components/SuperAdmins/List.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                <button
+                  className={styles.acceptBtn}
+                  onClick={() => handleSaveClick(superAdmin._id, email, pass)}
+                >
+                  Accept
+                </button>
+              ) : (
+                <button
+                  onClick={() => handleEditClick(superAdmin._id)}
+                  id={styles.editBtn}
```
Comment (Juanoauil) [inline src/Components/SuperAdmins/List.js:82]: Replace id by className, id is unique

File: src/Components/SuperAdmins/List.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                ></button>
+              )}
+            </td>
+            <td className={styles.tdBtn}>
+              {editUser === superAdmin._id ? (
+                <button className={styles.cancelBtn} onClick={() => handleCancelClick()}>
+                  Cancel
+                </button>
+              ) : (
+                <button id={styles.deleteBtn} onClick={() => deleteItem(superAdmin._id)}></button>
```
Comment (Juanoauil) [inline src/Components/SuperAdmins/List.js:92]: same here

**Review by SantiH97 — APPROVED** [review]
All looking good. Good work!!

**Comment by Juanoauil** [comment]
remove package-lock.json from changes

File: README.md [hunk 3]
```diff
@@ -2,43 +2,43 @@
...
 ### Install dependencies
 
     npm install
 
 ### Setup environment file
+
 create a file at root called `.env` and add this:
 
-    REACT_APP_API=<server url>
+    REACT_APP_API_URL=<server url>
```
Comment (Juanoauil) [inline README.md:13]: thank you

File: README.md [hunk 4]
```diff
@@ -2,43 +2,43 @@
...
 
-
-
 <br>
 
 ### License & Copyright
 
-© Radium Rocket "Become a Software Professional 2022"
\ No newline at end of file
+© Radium Rocket "Become a Software Professional 2022"
```
Comment (Juanoauil) [inline README.md:44]: Remove package-lock.json from your changes

File: src/Components/SuperAdmins/Form.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+  const submit = (e) => {
+    e.preventDefault();
+    create(email, pass);
+  };
+
+  return (
+    <div>
+      <h2 id={styles.titleForm}>Create a Super Admin</h2>
+      <form id={styles.formPost}>
```
Comment (Juanoauil) [inline src/Components/SuperAdmins/Form.js:17]: it should be a className

File: src/Components/SuperAdmins/Form.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        <div>
+          <label>Email: </label>
+          <input
+            id={styles.emailImput}
```
Comment (Juanoauil) [inline src/Components/SuperAdmins/Form.js:21]: same here

File: README.md (hunk 4 above)
Comment (Dannylez) [reply README.md:44]: there is no changes in package-lock.json, I had just commented something in an attempt to fix a bug, but I already keep it as it was originally.

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**